### ISSUE TITLE
wsltty (wslbridge2) segfaults with latest wsl version

### ISSUE THREAD

**Issue by blandcr** [mintty/wsltty#317]
Hi, the latest WSL update seems to be not compatible with the wslbridge2 that ships with wsltty. Manually updating to the v0.9 release of wslbridge2 (from https://github.com/Biswa96/wslbridge2 releases) seems to fix the problem.

WSL version info
```
PS C:\Users\fallacy> wsl --version
WSL version: 0.70.4.0
Kernel version: 5.15.68.1
WSLg version: 1.0.45
MSRDC version: 1.2.3575
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.674
```
wsltty invocation (via shortcut)
```
%LOCALAPPDATA%\wsltty\bin\mintty.exe --WSL="Arch" --configdir="%APPDATA%\wsltty" -~  -
```
If you want the wslbridge2 stackdump from the original segfault I can provide it but since just updating to the latest release seems to fix the problem I'm not sure it's needed. Anyway just thought yall might want to know and sorry if this isn't the right way to report the issue.

**Comment by mintty** [comment]
Released 3.6.1.2. Please retest.

**Comment by brc-intelight** [comment]
Works great now! Thanks!
